client/src/wh/logged_in/personalised_jobs/views.cljs

```diff
@@ -17,7 +29,11 @@
       (when (= type-of-jobs :recommended)
         [:div.has-bottom-margin
          [link [:button.button "Improve recommendations"] :improve-recommendations :class "level-item"]])]]
-    (let [parts (partition-all 3 (<sub [::subs/jobs]))
+    (let [{on-close :on-close
```

Comment:
I'd prefer the `:keys` method of destructuring in this instance because the keys and var names are/can be the same.